src/resources/elements/dealSummary/dealSummary.html

```diff
@@ -45,4 +45,4 @@ <h2>${deal.title}</h2>
     </div>
 
   </div>
-</template>
+</template>
```

Comment:
@giladt @dkent600 @paweljanicki @IonelLupu 
We should agree on adding new line or not

Add: 👍 
No New line 🚀

Reply:
I have no opinion yet. When/why does it matter?

Reply:
It matters, because otherwise every PR potentially adds/removes, thus creates a diff aka. noise for the review.

Reply:
We can also enforce that with eslint

Reply:
Yes, must be a rule in eslint

Reply:
actually already is a rule in our eslint config